src/Types/InputOutput/Scalar/IntType.hack

```diff
@@ -38,4 +38,14 @@ final class IntInputType extends ScalarInputType {
         );
         return $node->getRawValue();
     }
+
+    <<__Override>>
+    protected function serialize(int $value): int {
+        \Slack\GraphQL\assert(
+            $value >= self::MIN_SAFE_VALUE && $value <= self::MAX_SAFE_VALUE,
+            'Integers must be in 32-bit range, got %d',
+            $value,
+        );
```

Comment:
This assertion could be pulled into a private helper since it's used like 3 times in this class.